awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -19,32 +19,43 @@ class ModuleDocFragment(object):
     - If value not set, will try environment variable C(CONTROLLER_HOST) and then config files
     - If value not specified by any means, the value of C(127.0.0.1) will be used
     type: str
-    aliases: [ tower_host ]
+    aliases: [ tower_host, aap_hostname ]
   controller_username:
     description:
     - Username for your controller instance.
     - If value not set, will try environment variable C(CONTROLLER_USERNAME) and then config files
     type: str
-    aliases: [ tower_username ]
+    aliases: [ tower_username, aap_username ]
   controller_password:
     description:
     - Password for your controller instance.
     - If value not set, will try environment variable C(CONTROLLER_PASSWORD) and then config files
     type: str
-    aliases: [ tower_password ]
+    aliases: [ tower_password , aap_password ]
+  aap_token:
+    description:
+    - The OAuth token to use.
+    - This value can be in one of two formats.
+    - A string which is the token itself. (i.e. bqV5txm97wqJqtkxlMkhQz0pKhRMMX)
+    - A dictionary structure as returned by the token module.
+    - If value not set, will try environment variable C(CONTROLLER_OAUTH_TOKEN) and then config files
+    type: raw
+    version_added: "3.7.0"
+    aliases: [ tower_oauthtoken, controller_oauthtoken]
```

Comment:
Does this still make sense?

Reply:
agree, I think this is okay if it has no aliases, due to other recent changes